helm/solr-operator/crds/crds.yaml

```diff
@@ -1053,9 +1053,12 @@ spec:
                 type: object
               repositoryName:
                 description: The name of the repository to use for the backup.  Defaults to "legacy_local_repository" if not specified (the auto-configured repository for legacy singleton volumes).
+                pattern: '[a-zA-Z0-9]([-_a-zA-Z0-9]*[a-zA-Z0-9])?'
                 type: string
               solrCloud:
                 description: A reference to the SolrCloud to create a backup for
+                minLength: 63
```

Comment:
[-1/Q] I commented above on this value in `solr.apache.org_solrbackups.yaml`, suspecting it was something you'd set for testing but hadn't removed yet.  If that ends up being the case, I figured I'd comment here as a reminder to make sure that your change gets propagated to this file as well.